ceresdbproto_deps/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "ceresdbproto_deps"
+version = "0.1.0"
+authors = ["CeresDB Authors <ceresdbservice@gmail.com>"]
+edition = "2021"
```

Comment:
keep edition the same with other crates?

Reply:
Ok but 2021 edition has been adopted by some crates already. And we may ugprade edition of all the crates to 2021 in another PR.

Reply:
Opened https://github.com/CeresDB/ceresdb/issues/143